Header header
geometry_msgs/Pose target
geometry_msgs/Pose obstacle
bool has_obstacle
bool point_at_target
